### ISSUE TITLE
Staging Review finding: Heading structure for Blue Button form flow

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99393]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

This ticket was part of a merged Staging Review with MHV Medical Records [Download Records](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92340), [Flow/Images](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95081) and [Settings](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95082)

## Product Information
**Team:** vfs-mhv-medical-records
**Product:** MHV Medical Records
**Feature:** Phase 1 Radiology Flow/Images, Download Records, and Settings

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the forms library as currently coded, the H1 for each page is the name of the form, and an H2 is hard-coded into the step indicator component. That means any headings you add to a form page should start with an H3, beneath the step indicator H2 in the heading hierarchy.

Each page of the Blue Button form flow has an H2 that describes the purpose of the page, which doesn't fit this heading pattern.

<details><summary>Link, screenshot or steps to recreate</summary>

![Select date range screen with one input. The heading "Select date range" is an H2.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/92340-Brian-3.png?raw=true)

</details>

## Recommended action

Each of the H2s included in the form flow should be H3s.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by lichellebain** [comment]
https://jira.devops.va.gov/browse/MHV-65366

**Comment by bryan-riley-va** [comment]
Closed Sprint 149
